src/backend/env/mod.rs

```diff
@@ -200,6 +201,9 @@ pub struct State {
     #[serde(skip)]
     pub last_upgrade: u64,
 
+    #[serde(skip)]
+    pub token_fees_burned: Token,
```

Comment:
So now we will only update it on upgrades?

Reply:
Yes, I think this is good enough. This metric is not required to be real-time.